docs/brillig/00_intro.md

```diff
@@ -0,0 +1,47 @@
+# Intro to Brillig
+BRILLIG - Bytecode Runtime Interface for Liberated Language Implementation... G
+
+The G stands for whatever you like, like [Gabagool](https://en.wikipedia.org/wiki/Gabagool).
+
+Noir can compile to bytecode for Brillig, a general VM architecture for use with Noir.
+It is used for functions that are unconstrained, non-deterministic, or otherwise need a VM. 
+
+Why we need Brillig
+---
+
+Noir is a zero-knowledge language that specializes in zero-knowledge proofs that flexibly can compile code to constraints. Despite its specialization, Noir still requires a general VM for the following reasons:
+
+1) Unconstrained execution
+
+In Noir, not all execution situations are fully constrained.
```

Comment:
No "In Noir"

Maybe something like:
"Enabling a circuit language to perform unconstrained execution is a powerful tool. Being able to execute logic outside of a circuit is critical for both circuit performance and constructing proofs on information that is external to a circuit." 
Then go into the example of fetching encrypted notes. 

I have some ideas for this section that I will submit in a PR